kaldb/src/test/java/com/slack/kaldb/chunkManager/IndexingChunkManagerTest.java

```diff
@@ -312,8 +312,7 @@ public void testDeleteStaleDataDoesNothingWhenGivenLimitLessThan0()
 
     // Get the count of the amount of indices so that we can confirm we've cleaned them up
     // after the rollover
-    final File dataDirectory = new File(indexerConfig.getDataDirectory());
-    final File indexDirectory = new File(dataDirectory.getAbsolutePath() + "/indices");
+    final File indexDirectory = tmpPath.resolve("indices").toFile();
```

Comment:
`indexerConfig.getDataDirectory()` is not set so this resolved to `kaldb/indices` in your project directory which is not what the test uses to create these indices.

If you see ` initChunkManager` our chunk is created from this path `tmpPath.toFile().getAbsolutePath()`